Integrations/integration-CheckpointFirewall

```diff
@@ -0,0 +1,707 @@
+            sendRequest('publish', '{}');
+            res = JSON.parse(result.Body);
+            md += tblToMd('Blocked Destination IP Table', res);
+            temp.push(res);
+        }
+
+        context = {CheckpointFWRule : jsonToEntity(temp,entityDict.CheckpointFWRule)};
+
+        return {Type: entryTypes.note,
+            Contents: result,
+            ContentsFormat: formats.json,
+            HumanReadable: md,
+            EntryContext: context
+        };
+    };
+
+    var show_hosts = function(){
+        var context = {};
+        var body = {limit: args.limit, offset: args.offset, order: args.order};
+        var result = sendRequest('show-hosts', JSON.stringify(body));
+        var res = JSON.parse(result.Body);
+        var md = tblToMd('Hosts table', res.objects);
+        context.Endpoint = jsonToEntity(res.objects, entityDict.Endpoint);
+
+        return {Type: entryTypes.note,
+            Contents: result,
+            ContentsFormat: formats.json,
+            HumanReadable: md,
+            EntryContext: context
+        };
+
+    };
+
+    var task_status = function(){
+        var result = sendRequest('show-task', JSON.stringify({'task-id':args.task_id}));
+        var res =  JSON.parse(result.Body);
+
+        var md = tblToMd('Task Status Table',res.tasks);
+        var context = {CheckpointFWTask : jsonToEntity(res.tasks, entityDict.CheckpointFWTask)};
+
+        return {Type: entryTypes.note,
+            Contents: result,
+            ContentsFormat: formats.json,
+            HumanReadable: md,
+            EntryContext: context
+        };
+    };
+
+    var set_rule = function(){
+        var result;
+        if(args.uid || args.name || args.rule_number){
+            var body = {
+                uid : args.uid,
+                name: args.name,
+                'rule-number': args.rule_number,
+                layer: args.layer,
+                enabled: args.enabled
+            };
+            result = sendRequest('set-access-rule', JSON.stringify(body));
+            sendRequest('publish', '{}');
+        }
+        else{
+            logout();
+            throw 'Set rule requires at least one of the following arguments - uid, name, rule_number';
+        }
+
+        var res =  JSON.parse(result.Body);
+        var md = tblToMd('Set Rule', res);
+        var context = {CheckpointFWRule : jsonToEntity(res, entityDict.CheckpointFWRule)};
+
+        return {Type: entryTypes.note,
+            Contents: result,
+            ContentsFormat: formats.json,
+            HumanReadable: md,
+            EntryContext: context
+        };
+    };
+
+    var delete_rule = function(){
+        if(args.uid || args.name || args.rule_number){
+            var body = {
+                uid : args.uid,
+                name: args.name,
+                'rule-number': args.rule_number,
+                layer: args.layer
+            };
+            res = sendRequest('delete-access-rule', JSON.stringify(body));
```

Comment:
you can use `var res = ...` instead and move `return JSON.parse(res.Body);` inside the body
(we should try to avoid global variables - without `var` declaration)